tools/clang/test/HLSLFileCheck/hlsl/objects/CbufferLegacy/cbufferSize3.hlsl

```diff
@@ -0,0 +1,13 @@
+// RUN: %dxc -E main -T ps_6_0 %s | FileCheck %s
+
+// CHECK: error: CBuffer size is 65540 bytes, exceeding maximum of 65536 bytes.
+
+cbuffer Foo1 : register(b5)
+{
+  float arr[4097];
```

Comment:
are all these entries converted into float4 somehow? Otherwise, I don't see how this exceeds the limit.

Reply:
Yes, arrays elements in legacy cbuffer format are aligned to 16-byte rows (4 x 32-bit values)